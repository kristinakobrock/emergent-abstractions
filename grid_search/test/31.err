  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 27.77it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 24.92it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 24.08it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 27.48it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 29.46it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 24.01it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 23.22it/s] 52%|###############################################8                                            | 26/50 [00:01<00:01, 21.42it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 23.60it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 23.92it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 22.55it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 22.36it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 21.97it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 23.71it/s] 96%|########################################################################################3   | 48/50 [00:02<00:00, 23.20it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.10it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.49it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 29.78it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 27.62it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 26.80it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.61it/s]
