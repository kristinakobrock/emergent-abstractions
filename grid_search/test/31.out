# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7036333084106445, "acc": 0.5830994844436646, "length": 2.1922805309295654, "mode": "train", "epoch": 1}
{"loss": 0.6649919748306274, "acc": 0.6298611164093018, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6642783880233765, "acc": 0.6223912835121155, "length": 1.2350279092788696, "mode": "train", "epoch": 2}
{"loss": 0.6604763269424438, "acc": 0.6314236521720886, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.662045955657959, "acc": 0.6224753856658936, "length": 1.2172969579696655, "mode": "train", "epoch": 3}
{"loss": 0.6613878607749939, "acc": 0.6315972805023193, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6423104405403137, "acc": 0.6384934186935425, "length": 2.043489694595337, "mode": "train", "epoch": 4}
{"loss": 0.6264388561248779, "acc": 0.6627549529075623, "length": 2.3333332538604736, "mode": "test", "epoch": 4}
{"loss": 0.6143289804458618, "acc": 0.6669455766677856, "length": 2.200077772140503, "mode": "train", "epoch": 5}
{"loss": 0.6315882802009583, "acc": 0.6452311873435974, "length": 1.9444444179534912, "mode": "test", "epoch": 5}
{"loss": 0.5895362496376038, "acc": 0.6899603605270386, "length": 2.431536912918091, "mode": "train", "epoch": 6}
{"loss": 0.59663987159729, "acc": 0.7041015625, "length": 3.0208332538604736, "mode": "test", "epoch": 6}
{"loss": 0.5516640543937683, "acc": 0.7223709225654602, "length": 2.742687463760376, "mode": "train", "epoch": 7}
{"loss": 0.5883755087852478, "acc": 0.7082628011703491, "length": 2.9583332538604736, "mode": "test", "epoch": 7}
{"loss": 0.5085605382919312, "acc": 0.7570962905883789, "length": 2.920168161392212, "mode": "train", "epoch": 8}
{"loss": 0.5979467630386353, "acc": 0.7286295890808105, "length": 3.0729167461395264, "mode": "test", "epoch": 8}
{"loss": 0.4814564883708954, "acc": 0.774290144443512, "length": 3.094461441040039, "mode": "train", "epoch": 9}
{"loss": 0.6370275616645813, "acc": 0.7144965529441833, "length": 3.4270832538604736, "mode": "test", "epoch": 9}
{"loss": 0.45377323031425476, "acc": 0.7861159443855286, "length": 3.1984615325927734, "mode": "train", "epoch": 10}
{"loss": 0.560428261756897, "acc": 0.7477104663848877, "length": 3.53125, "mode": "test", "epoch": 10}
{"loss": 0.4284934997558594, "acc": 0.8111177086830139, "length": 3.3225626945495605, "mode": "train", "epoch": 11}
{"loss": 0.5567121505737305, "acc": 0.7688043117523193, "length": 3.3888888359069824, "mode": "test", "epoch": 11}
{"loss": 0.39273911714553833, "acc": 0.8300220966339111, "length": 3.422208309173584, "mode": "train", "epoch": 12}
{"loss": 0.48982539772987366, "acc": 0.7876248359680176, "length": 3.6145832538604736, "mode": "test", "epoch": 12}
{"loss": 0.36040401458740234, "acc": 0.8460537195205688, "length": 3.658015489578247, "mode": "train", "epoch": 13}
{"loss": 0.4963611960411072, "acc": 0.7970215678215027, "length": 3.7048611640930176, "mode": "test", "epoch": 13}
{"loss": 0.3380657136440277, "acc": 0.8589394092559814, "length": 3.7094883918762207, "mode": "train", "epoch": 14}
{"loss": 0.4524095356464386, "acc": 0.8132052421569824, "length": 3.784722328186035, "mode": "test", "epoch": 14}
{"loss": 0.3134584128856659, "acc": 0.8732341527938843, "length": 3.7457382678985596, "mode": "train", "epoch": 15}
{"loss": 0.40354764461517334, "acc": 0.8444769382476807, "length": 3.9236111640930176, "mode": "test", "epoch": 15}
{"loss": 0.2928505837917328, "acc": 0.8840702772140503, "length": 3.8180742263793945, "mode": "train", "epoch": 16}
{"loss": 0.43237176537513733, "acc": 0.8371527194976807, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.27241963148117065, "acc": 0.8948267102241516, "length": 3.9063422679901123, "mode": "train", "epoch": 17}
{"loss": 0.3836623430252075, "acc": 0.8529513478279114, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.25053897500038147, "acc": 0.9047386646270752, "length": 3.95259690284729, "mode": "train", "epoch": 18}
{"loss": 0.37184977531433105, "acc": 0.8642361164093018, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.23143123090267181, "acc": 0.9145411849021912, "length": 3.9736475944519043, "mode": "train", "epoch": 19}
{"loss": 0.37455570697784424, "acc": 0.8687499165534973, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.22725574672222137, "acc": 0.9190831184387207, "length": 3.975065231323242, "mode": "train", "epoch": 20}
{"loss": 0.3603534698486328, "acc": 0.8659722805023193, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.2063019722700119, "acc": 0.9273119568824768, "length": 3.9786946773529053, "mode": "train", "epoch": 21}
{"loss": 0.32432231307029724, "acc": 0.8880208134651184, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.19126202166080475, "acc": 0.9360510110855103, "length": 3.9857900142669678, "mode": "train", "epoch": 22}
{"loss": 0.3709539473056793, "acc": 0.8821181058883667, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.1821237951517105, "acc": 0.9400099515914917, "length": 3.9858274459838867, "mode": "train", "epoch": 23}
{"loss": 0.3064948320388794, "acc": 0.902256965637207, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.1679365485906601, "acc": 0.9454479217529297, "length": 3.9891490936279297, "mode": "train", "epoch": 24}
{"loss": 0.3040536642074585, "acc": 0.9029514789581299, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.16728977859020233, "acc": 0.9463015794754028, "length": 3.991173505783081, "mode": "train", "epoch": 25}
{"loss": 0.31789520382881165, "acc": 0.9046875834465027, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.16479364037513733, "acc": 0.9453220367431641, "length": 3.9904298782348633, "mode": "train", "epoch": 26}
{"loss": 0.3396199643611908, "acc": 0.8927083611488342, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.16306832432746887, "acc": 0.9468523263931274, "length": 3.991607666015625, "mode": "train", "epoch": 27}
{"loss": 0.2878817915916443, "acc": 0.9121527075767517, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.15450255572795868, "acc": 0.9501768946647644, "length": 3.9932427406311035, "mode": "train", "epoch": 28}
{"loss": 0.32601773738861084, "acc": 0.9041667580604553, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.14699140191078186, "acc": 0.9540697932243347, "length": 3.9936654567718506, "mode": "train", "epoch": 29}
{"loss": 0.3101596236228943, "acc": 0.9008679986000061, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.14805904030799866, "acc": 0.9511595964431763, "length": 3.9946765899658203, "mode": "train", "epoch": 30}
{"loss": 0.3368076980113983, "acc": 0.9013888835906982, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.14628691971302032, "acc": 0.9517174959182739, "length": 3.9959356784820557, "mode": "train", "epoch": 31}
{"loss": 0.27638211846351624, "acc": 0.9175347089767456, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.1417267918586731, "acc": 0.9551584124565125, "length": 3.995781421661377, "mode": "train", "epoch": 32}
{"loss": 0.26059484481811523, "acc": 0.925694465637207, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.13385120034217834, "acc": 0.9561437964439392, "length": 3.9966471195220947, "mode": "train", "epoch": 33}
{"loss": 0.3115161657333374, "acc": 0.9081597924232483, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.13309170305728912, "acc": 0.9563180208206177, "length": 3.9954683780670166, "mode": "train", "epoch": 34}
{"loss": 0.3069498538970947, "acc": 0.9090278148651123, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.13543736934661865, "acc": 0.9546710848808289, "length": 3.9963808059692383, "mode": "train", "epoch": 35}
{"loss": 0.3090754747390747, "acc": 0.9067708253860474, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.13375182449817657, "acc": 0.9580605626106262, "length": 3.997030735015869, "mode": "train", "epoch": 36}
{"loss": 0.29626429080963135, "acc": 0.9097222089767456, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.13862350583076477, "acc": 0.9567848443984985, "length": 3.9966330528259277, "mode": "train", "epoch": 37}
{"loss": 0.382473349571228, "acc": 0.8923611044883728, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.13261307775974274, "acc": 0.9563460946083069, "length": 3.997534990310669, "mode": "train", "epoch": 38}
{"loss": 0.3303065001964569, "acc": 0.9026041030883789, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.11917438358068466, "acc": 0.9594074487686157, "length": 3.9983458518981934, "mode": "train", "epoch": 39}
{"loss": 0.3397050201892853, "acc": 0.9055556058883667, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.12973031401634216, "acc": 0.9571046233177185, "length": 3.9982595443725586, "mode": "train", "epoch": 40}
{"loss": 0.3349248468875885, "acc": 0.9041666388511658, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.13987553119659424, "acc": 0.9529030323028564, "length": 3.9982097148895264, "mode": "train", "epoch": 41}
{"loss": 0.301458477973938, "acc": 0.9152777791023254, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.1241452768445015, "acc": 0.958311915397644, "length": 3.9980390071868896, "mode": "train", "epoch": 42}
{"loss": 0.31820178031921387, "acc": 0.9130207896232605, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.12223052978515625, "acc": 0.958891749382019, "length": 3.9983930587768555, "mode": "train", "epoch": 43}
{"loss": 0.29818350076675415, "acc": 0.9131944179534912, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.11945168673992157, "acc": 0.9599835276603699, "length": 3.9982962608337402, "mode": "train", "epoch": 44}
{"loss": 0.29592955112457275, "acc": 0.9126736521720886, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.11687289923429489, "acc": 0.961024284362793, "length": 3.9989054203033447, "mode": "train", "epoch": 45}
{"loss": 0.3120947778224945, "acc": 0.9131944179534912, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.11728080362081528, "acc": 0.9609745144844055, "length": 3.999229669570923, "mode": "train", "epoch": 46}
{"loss": 0.30473795533180237, "acc": 0.9168403148651123, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.11276302486658096, "acc": 0.962641716003418, "length": 3.999201774597168, "mode": "train", "epoch": 47}
{"loss": 0.29671353101730347, "acc": 0.9149305820465088, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.10464991629123688, "acc": 0.9640853404998779, "length": 3.9994277954101562, "mode": "train", "epoch": 48}
{"loss": 0.3231624662876129, "acc": 0.9085069298744202, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.1084643006324768, "acc": 0.9624701142311096, "length": 3.9992527961730957, "mode": "train", "epoch": 49}
{"loss": 0.32112735509872437, "acc": 0.9086804986000061, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.10957704484462738, "acc": 0.9622470736503601, "length": 3.9995369911193848, "mode": "train", "epoch": 50}
{"loss": 0.3150731325149536, "acc": 0.9135415554046631, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.1085212230682373, "acc": 0.9626481533050537, "length": 3.999459981918335, "mode": "train", "epoch": 51}
{"loss": 0.3155837655067444, "acc": 0.9109375476837158, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.1087203174829483, "acc": 0.9638010263442993, "length": 3.998926877975464, "mode": "train", "epoch": 52}
{"loss": 0.3120023310184479, "acc": 0.9147568941116333, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.1034620851278305, "acc": 0.9653529524803162, "length": 3.9995334148406982, "mode": "train", "epoch": 53}
{"loss": 0.3119242489337921, "acc": 0.9104166030883789, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.10340678691864014, "acc": 0.9641987681388855, "length": 3.9993784427642822, "mode": "train", "epoch": 54}
{"loss": 0.314410537481308, "acc": 0.9166666865348816, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.10838593542575836, "acc": 0.9635016918182373, "length": 3.9991543292999268, "mode": "train", "epoch": 55}
{"loss": 0.29373013973236084, "acc": 0.9208332896232605, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.11626272648572922, "acc": 0.9612635374069214, "length": 3.9991064071655273, "mode": "train", "epoch": 56}
{"loss": 0.35490095615386963, "acc": 0.9067708253860474, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.12574343383312225, "acc": 0.9557446837425232, "length": 3.999480724334717, "mode": "train", "epoch": 57}
{"loss": 0.3193803131580353, "acc": 0.9038195013999939, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.11983281373977661, "acc": 0.9596570730209351, "length": 3.9994757175445557, "mode": "train", "epoch": 58}
{"loss": 0.3253658413887024, "acc": 0.9081597924232483, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.11324433237314224, "acc": 0.9610397815704346, "length": 3.999587297439575, "mode": "train", "epoch": 59}
{"loss": 0.35902369022369385, "acc": 0.8982638120651245, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.12207435071468353, "acc": 0.9578156471252441, "length": 3.999316692352295, "mode": "train", "epoch": 60}
{"loss": 0.3510403335094452, "acc": 0.9020833969116211, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.8230355978012085
